cmd/osm-bootstrap/osm-bootstrap.go

```diff
@@ -203,48 +244,9 @@ func main() {
 		metricsstore.DefaultMetricsStore.ErrCodeCounter,
 		metricsstore.DefaultMetricsStore.HTTPResponseTotal,
```

Comment:
As mentioned on the call, can you bring the reconciler to above the httpServer, and log fatal on an error starting that?